src/pages/settings/AppDownloadLinks.tsx

```diff
@@ -61,6 +65,18 @@ function AppDownloadLinksPage() {
                 title={translate('initialSettingsPage.aboutPage.appDownloadLinks')}
                 onBackButtonPress={() => Navigation.goBack()}
             />
+
+            <QRShareWithDownload
```

Comment:
We should still use `QRShare`. I don't see any reason why not to? We don't need the neither ref nor the download functionality